marlowe/src/Language/Marlowe/Semantics.hs

```diff
@@ -157,16 +157,17 @@ newtype ValueId = ValueId ByteString
 
     Values can also be scaled, and combined using addition, subtraction, and negation.
 -}
-data Value = AvailableMoney AccountId Token
+data Value a = AvailableMoney AccountId Token
```

Comment:
do we use this parameter anywhere? i.e. isn't it always `Observation`?

Reply:
@shmish111 Plutus doesn't support mutually recursive datatypes, basically you had same problem for `Case` before solved in exactly same way

Reply:
ah I see 👍